guide/src/describing/connections.md

```diff
@@ -5,12 +5,12 @@ of a compatible [type](types.md):
 * `from` = [IO Reference](io_references.md) to the Data source that this connection comes from
 * `to` = [IO Reference](io_references.md) to a Data sink that this connection goes to
 
-The type of a data flow is inferred from the two IOs that it connects (after checking they coincide)
+The type of a data flow is inferred from the two IOs that it connects (after checking they coincide to each other)
```

Comment:
"with each other" would be perfect English!
or "checking the types match" maybe better! Thanks